lib/mcb/commands/users/list.rb

```diff
@@ -0,0 +1,16 @@
+summary 'List users in the tb'
+
+run do |opts, args, _cmd|
+  MCB.init_rails(opts)
+
+  users = if args.any?
+            User.where(id: args.to_a)
```

Comment:
Do we want one of these for dfe sign-in id? Maybe also by email address?

Reply:
Ya, I think that those would be nice to haves, although I'd be more likely to add that to the `show` command since I imagine that's more likely to be used to pull out individual users. At the very least maybe we should be exposing the DFE sign-in ids here.

Reply:
Quite right, I meant the `show` command.